test/integration/standard_filter_test.rb

```diff
@@ -358,6 +358,12 @@ def test_append
     assert_template_result('bcd',"{{ a | append: b}}",assigns)
   end
 
+  def test_concat
+    assert_equal([1,2,3,4 ], @filters.concat([1,2], [3,4] ))
+    assert_equal([1,2,'a' ], @filters.concat([1,2], ['a'] ))
+    assert_equal([1,2, 10 ], @filters.concat([1,2], [ 10] ))
+  end
+
```

Comment:
@parkr @fw42 what do you think now? will works only with arrays, and rise exceptions with other types.